hhs_oauth_server/settings/aws-dev.py

```diff
@@ -44,9 +44,22 @@
 DEV_SPECIFIC_APPS = [
     # Installation/Site Specific apps based on  -----------------
     'storages',
+    # A test client - moved to aws-test / dev /impl settings
+    'apps.testclient',
```

Comment:
This is one of those things that would be fine if we weren't using our dev environment for our preview. For the moment, dev is essentially our highest environment though, and needs to be treated as such. Hopefully we can fix this when we move to the new URLs.